selftests/check.py

```diff
@@ -201,6 +201,22 @@ def parse_args():
     parser.add_argument('--disable-plugin-checks',
                         help='Disable checks for a plugin (by directory name)',
                         action='append', default=[])
+    parser.add_argument('--disable-selftests-nrunner',
```

Comment:
We need a better naming here. I had the impression it was disabling all the `nrunner` tests, but that is not true.

Reply:
Good point. I'm going to rename it to `--disable-selftests-nrunner-interface`,